internal/server/routes.go

```diff
@@ -184,6 +192,8 @@ func add[Req, Res any](a *API, group *routeGroup, route route[Req, Res]) {
 	bindRoute(a, group.RouterGroup, route.method, route.path, handler)
 }
 
+var redirectResponseType = reflect.TypeOf(api.RedirectResponse{})
```

Comment:
I think the file that added `api.RedirectResponse` did not get checked in. I'm not seeing it in this branch.